pkg/build/builder/source.go

```diff
@@ -387,7 +387,7 @@ func extractSourceFromImage(ctx context.Context, dockerClient DockerClient, stor
 
 	if auths != nil {
 		for registry, ac := range auths.Configs {
-			glog.Infof("Setting authentication for registry %q using %#v", registry, ac)
+			glog.Infof("Setting authentication for registry %q using %q.", registry, ac.ServerAddress)
```

Comment:
Log level should be V(5) for this (debug). - `glog.V(5).Infof(...)`
There are similar log statements to this which should also be V(5):

https://github.com/openshift/builder/blob/master/pkg/build/builder/daemonless.go#L55-L60
https://github.com/openshift/builder/blob/master/pkg/build/builder/daemonless.go#L102-L109

Reply:
@adambkaplan fixed